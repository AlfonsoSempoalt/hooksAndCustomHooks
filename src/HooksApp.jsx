import React from 'react'
import PropTypes from 'prop-types'

const HooksApp = props => {
    return (
        <>
         <h1>Hello world</h1>   
        </>
    )
} 

HooksApp.propTypes = {

}

export default HooksApp
