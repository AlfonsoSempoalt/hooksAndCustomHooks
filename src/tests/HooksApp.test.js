import { expect, describe } from '@jest/globals';
import React from 'react';
import '@testing-library/jest-dom';
import { shallow } from 'enzyme';
import HooksApp from '../HooksApp';

describe('Testing to <HooksApp />', () => {
  
  
  test('should match with the snapshot', () => {
    const wrapper = shallow(<HooksApp />);
    expect(wrapper).toMatchSnapshot();
  });
});
