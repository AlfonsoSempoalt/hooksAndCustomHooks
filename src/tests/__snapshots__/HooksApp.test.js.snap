// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing to <HooksApp /> should match with the snapshot 1`] = `
<Fragment>
  <h1>
    Hello world
  </h1>
</Fragment>
`;
